openedx/features/course_experience/tests/views/test_course_home.py

```diff
@@ -109,3 +119,47 @@ def test_queries(self):
             with check_mongo_calls(4):
                 url = course_home_url(self.course)
                 self.client.get(url)
+
+
+@ddt.ddt
+class TestCourseHomePageAccess(CourseHomePageTestCase):
+    """
+    Test access to the course home page.
+    """
+    @override_waffle_flag(UNIFIED_COURSE_TAB_FLAG, active=True)
```

Comment:
It seems like a very simple test that would verify that this flag is doing what we need it to do.  If it is complicated, you could skip.